debian/changelog

```diff
@@ -1,3 +1,10 @@
+cloud-init (20.2-45-g5f7825e2-0ubuntu1~16.04.2) UNRELEASED; urgency=medium
```

Comment:
Given that we'll be making this change across all releases, perhaps `ubuntu2~16.04.1` would be more apropos?

Reply:
Wait what. I thought you will push out cloud-init of all the things.

If you are publishing this as the version above, it needs to match in the postinst. below.

Reply:
This is not xenial only. This is xenial and up...... right? you are preparing uploads for xenial, bionic, focal and groovy right?

Reply:
As this is a packaging-only change, it's not easier from a process POV to do this as full backports: we still have to apply the packaging change to each branch. And assuming that we'd like to get this fix out sooner rather than later, uploading it by itself gives us a much narrower set of testing that we need to perform before we can release it to `-updates`.

Reply:
(And yes, I will prepare uploads for everywhere: I generally prefer to start in the oldest release, to discover any unexpected behaviour in the place we're most likely to find it.)